composer/datasets/streaming/download.py

```diff
@@ -82,26 +67,16 @@ def download_from_sftp(remote: str, local: str) -> None:
     # Default port
     port = port if port else 22
 
-    # Local tmp
-    local_tmp = local + '.tmp'
-    if os.path.exists(local_tmp):
-        os.remove(local_tmp)
+    object_store = SFTPObjectStore(
```

Comment:
Is it possible to incorporate the timeout into the SFTP object store? No worries if not...